server/adapters/snowflake.go

```diff
@@ -53,7 +57,7 @@ type SnowflakeConfig struct {
 }
 
 //Validate required fields in SnowflakeConfig
-func (sc *SnowflakeConfig) Validate() error {
+func (sc *SnowflakeConfig) Validate(strict bool) error {
```

Comment:
Same. Schema and Password aren't required

Reply:
Reverted